config/settings/common.py

```diff
@@ -222,15 +224,16 @@
 CELERY_RESULT_SERIALIZER = 'seed_json'
 CELERY_TASK_RESULT_EXPIRES = 18000  # 5 hours
 
+CELERY_IMPORTS = ('seed.energy.meter_data_processor.tasks')
 CELERYBEAT_SCHEDULE = {
-    'Run daily': {
-        'task': 'seed.energy.tasks.aggregate_monthly_data',
-        'schedule': timedelta(seconds=300), # For Demo purpose, should be days=1 in product environment
+    'Run monthly': {
+        'task': 'seed.energy.meter_data_processor.tasks.aggregate_monthly_data',
```

Comment:
I think we have moved this file a couple times. I think the tasks.py file for the energy 'app' is in /seed/energy/tasks.py